src/lib/contents/codespaces.ts

```diff
@@ -10,6 +10,58 @@ import type {
   FeatureTableToc,
 } from "../components/ui-library/feature-table/feature-table";
 
+import type { Testimonial } from "$lib/types/testimonial";
+
+export const testimonials: Testimonial[] = [
+  {
+    name: "Alejandro Sánchez",
+    avatar: "alejandro-sanchez.jpeg",
+    org: `
+      <span>
+        4GeeksAcademy
+      </span>
+    `,
+    role: "Founder of",
+    text: `
+        <p>
+        We have tested Codespaces thoroughly this week, and it was more complex to configure than Gitpod, it's harder to use for students, and documentation is way worse.
+        </p>
+      `,
+  },
+  /**
+   * Avatar image taken from: https://github.com/soup-bowl
+   * Blog post: https://soupbowl.blog/2023/02/a-year-of-cloud-code-in-review
+   */
+  {
+    name: "Casey LP",
+    avatar: "casey-lp.jpeg",
+    org: ``,
+    role: "DevOps Engineer",
```

Comment:
```suggestion
    role: "DevOps Engineer and Author at",
```

Reply:
It is their personal blog. So, How about `Author of Soupbowl.blog` ?
```suggestion
    role: "DevOps Engineer and Author of",
```